### ISSUE TITLE
What is the failing badge at the top of the readme?

### ISSUE THREAD

**Issue by pdeffebach** [JuliaData/DataFramesMeta.jl#200]
At the top of the readme is a failing badge with markdown links

```
[![DataFramesMeta](http://pkg.julialang.org/badges/DataFramesMeta_0.6.svg)](http://pkg.julialang.org/?pkg=DataFramesMeta?pkg=DataFramesMeta&ver=0.6)
```

What is this? Can I delete it? I think the line was last edited in 2017. 

cc @nalimilan

**Comment by nalimilan** [comment]
These badges used to reflect the testing status on pkg.julialang.org. But it's long been dead.

**Comment by pdeffebach** [comment]
Closed via #201
